Bottom: 219d8de826d5edff834507c28119e106aac444c2
Top:    b7eb65c699b80ebcd4fe32035605c44cc9ace9a6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 13:06:44 +0000

Avoid clearing pins / vias from polygons... it is already done!









---

diff --git a/src/draw.c b/src/draw.c
index 60e1175..9420bb1 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1314,6 +1314,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1336,6 +1337,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
